docs/how-to/rollback-blocks.md

```diff
@@ -0,0 +1,39 @@
+# How to Rollback Blocks
+
+It's possible when some broken data appear on the blockchain. In this case we
+have to rollback some blocks. Tool `cardano-auxx` provides such a functionality.
+
+First of all, we have to activate `repl`-mode:
+
+```
+$ stack exec -- cardano-auxx            \
```

Comment:
1. Add flags `--configuration-file configuration.yaml`, `--configuration-key mainnet_full`
2. Provide commands to use via `cmd`, not `repl`